util/util.go

```diff
@@ -132,3 +133,28 @@ func GenerateRandomECSPassword() string {
 	return string(s)
 
 }
+
+func WaitForSignal(attempts AttemptStrategy,api func() (bool,error)) error{
+
+	for attempt := attempts.Start(); attempt.Next(); {
+		needStop,err := api()
+
+		if(err != nil) {
+			return errors.New("execution failed")
+		}
+		if(needStop){
+			return nil;
+		}
+
+		if attempt.HasNext() {
+			continue;
+		}
+
+		if(needStop){
+			return nil;
+		} else {
+			return errors.New("timeout execution ")
+		}
+	}
+	panic("unreachable")
```

Comment:
`panic("unreachable")` is deprecated
see https://github.com/golang/go/issues/65